ibis/backends/flink/registry.py

```diff
@@ -319,6 +319,64 @@ def _string_to_timestamp(
     return f"TO_TIMESTAMP({arg}, {format_string})"
 
 
+def _time(translator: ExprTranslator, op: ops.temporal.Time) -> str:
+    from ibis.expr.datatypes.core import Timestamp
+
+    if isinstance(op.arg.dtype, Timestamp):
```

Comment:
Use the `is_timestamp()` method:

```suggestion
    if op.arg.dtype.is_timestamp():
```

Reply:
Thanks, fixed.